MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -3770,6 +3794,34 @@ public boolean newDay() {
         return true;
     }
 
+    private void processUnitReputation() {
```

Comment:
This function seems to deal primarily with crime score and pirates; is this function name accurate?

Reply:
When I wrote the function I anticipated it being needed for more than just crime. Good catch, I'll get it updated shortly :)